samples/wifi/raw_tx_packet/Kconfig

```diff
@@ -120,25 +120,25 @@ config RAW_TX_PACKET_SAMPLE_QUEUE_NUM
 
 choice
 	prompt "Select transmission mode"
-	default RAW_TX_PACKET_SAMPLE_TRANSMISSION_MODE_CONTINUOUS
+	default RAW_TX_PKT_SAMPLE_TRANSMISSION_MODE_CONTINUOUS
```

Comment:
(Generic comment for all instances where `TRANSMISSION` is used) 

Is `TRANSMISSION` necessary here (as in isn't it implied by `RAW_TX`) ? If you think it is necessary maybe change it to `TX`.